.github/workflows/winget.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish on Windows Package Manager
+
+on:
+  release:
+    types: [published]
```

Comment:
Is this going to catch a pre-release being marked as stable, or do you need `[released]` for that? (The [docs](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release) are a bit unclear to me on this point.)

Reply:
It won't be triggered when pre-release is changing to release. Usually, pre-release and its subsequent release have different versions, meaning they are different publications.

Reply:
I guess this can vary from one project to another, but it's not the case for clangd. Our stable releases typically start as pre-releases, and are then upgraded to stable. For example, https://github.com/clangd/clangd/releases/tag/21.1.0 is currently marked pre-release, but will subsequently be marked stable.